app/Mage.php

```diff
@@ -893,16 +893,16 @@ public static function getIsDeveloperMode()
      */
     public static function printException(Exception $e, $extra = '')
     {
-        if (self::$_isDeveloperMode) {
-            print '<pre>';
+        if (self::$_isDeveloperMode || PHP_SAPI == "cli") {
+            PHP_SAPI == "cli" or print '<pre>';
```

Comment:
I'm *very* much opposed to this style of programming. Please use `if` statements.

Reply:
also it changes behavior to always print on CLI even if the developer mode is not enabled

Reply:
Good point. I'd say that printing an exception while in CLI should be the default personally, but changing the behaviour in this case is undesirable I think. If the PR is limited to just removing the HTML output when using the CLI, then it's much less of a change overall.

Reply:
That's intentional because running report.php when on CLI is overkill in my opinion. The error info is already right there on your console and can easily be captured or the cron service will capture it and email it to you.

Reply:
sadly I dont live in this nice world, where I can expect cron jobs to send mails to me. And as long as you dont receive mails, you dont notice something is broken. So please dont make my life more complicated.

 And some may also have monitoring based on the fact, that this report file gets called.

Reply:
AFAIK all versions of Linux cron support the "MAILTO" option... Anyway, I'll consider the concerns given here and update at some point.